packages/keyring-controller/CHANGELOG.md

```diff
@@ -7,8 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.1.0]
+
 ### Changed
 
+- Add support for overwriting built-in keyring builders for the Simple and HD keyring ([#4362](https://github.com/MetaMask/core/pull/4362))
```

Comment:
Should this be in the "Added" category? This feels like a new feature to me (and would go with bumping this package by a minor)

Reply:
Yup, sounds good. We already are bumping the minor for the `KeyringController` so nothing to change for this. Done here: [16aed63](https://github.com/MetaMask/core/pull/4413/commits/16aed634b163794442d1f2d239ac9b875b024122)